src/Classes/Controllers/AddContactController.php

```diff
@@ -27,20 +27,20 @@ public function __invoke(Request $request, Response $response, array $args)
         $data = [
             'success' => false,
             'message' => 'Error!',
-            'data' => []
+            'data' => $newContact
```

Comment:
Maybe change this back to the empty array for consistency, but it doesn't do any harm like this.